File: all/abc217/abc217_b/abc217_b.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String[] s = new String[3];
    for(int i = 0; i < 3; i++) s[i] = stdIn.next();

    String[] ptn = {"ABC", "ARC", "AGC", "AHC"};
    boolean[] check = new boolean[4];
    for(int i = 0; i < 4; i++) check[i] = false;

    for(int i = 0; i < 3; i++) {
      for(int j = 0; j < 4; j++) {
        if(s[i].equals(ptn[j])) {
          check[j] = true;
          break;
        }
      }
    }

    for(int i = 0; i < 4; i++) {
      if(!check[i]) {
        System.out.println(ptn[i]);
        break;
      }
    }
  }
}

File: all/abc217/abc217_b/abc217_b.py
s = []
for _ in range(3):
  s.append(input())
s = sorted(s)

if s == ["ABC", "AGC", "AHC"]:
  print("ARC")
elif s == ["ABC", "AGC", "ARC"]:
  print("AHC")
elif s == ["ABC", "AHC", "ARC"]:
  print("AGC")
else:
  print("ABC")